src/components/SettingsGrid/SettingsGridCard/SettingsGridCard.js

```diff
@@ -14,41 +14,59 @@ type Props = {|
   +iconColor: JIconColor,
 |}
 
-class SettingsCard extends PureComponent<Props, *> {
+const isExternalURL = (url: string): boolean => /^https?:\/\//.test(url)
+
+class SettingsGridCard extends PureComponent<Props, *> {
   static defaultProps = {
     iconColor: 'blue',
   }
 
-  render() {
+  renderContent = () => {
     const {
-      iconName,
-      iconColor,
-      path,
       title,
       description,
+      iconColor,
+      iconName,
     } = this.props
 
     return (
-      <Link to={path} className='settings-card'>
-        {iconName &&
-        <div className='icon'>
+      <Fragment>
+        <div key='icon' className='icon'>
           <JIcon
             color={iconColor}
             name={iconName}
           />
-        </div>}
-        <div className='summary'>
+        </div>
+        <div key='summary' className='summary'>
           <div className='title'>
             <JText value={title} color='dark' size='header' />
           </div>
-          {description &&
+          { description &&
           <div className='description'>
             <JText value={description} color='dusk' />
-          </div>}
+          </div> }
         </div>
```

Comment:
let's do not use spaces after/before curly brackets in jsx, because a lot of code written in such style

Reply:
Maybe they have rule for this in eslint?

Reply:
Im pretty sure that such eslint rule exists

Reply:
I guess, the rule applying only for javascript parts, not for jsx.